public/js/pages/common.js

```diff
@@ -1,8 +1,9 @@
 //var domainUrl = window.location.origin
 //domainUrl = domainUrl.replace("8081", "1234");
-var domainUrl = 'http://localhost:1234';
+var domainUrl = 'http://' + window.location.host;
 var commonData = {
     "Preference": {
         "InfoEmail": "bhushan2250@gmail.com"
     }
-};
\ No newline at end of file
+}
+var stripePublisherKey = "pk_test_ULZHqL27xD6Zd0j3gnro1eY6";
```

Comment:
This public use for only generate token for process credit card, the credit card process can be completed by using only secrete key. This secret key is inside our config.js which is secured.